bin/todo

```diff
@@ -18,17 +18,21 @@ import re
 import sys
 import shutil
 
+# python2 compat
+if sys.version_info[0] == 2:
+    from future.builtins.misc import input
+
 import todo_tools
 
 
 def main():
     args = get_args()
 
-    # python 2 compatibility
-    if sys.version_info[0] == 2:
-        input = raw_input
-
-    if args.check:
+    if args.clear_todos:
+        sure = input("Are you sure you want to clear todo file? (y/N) ").lower()
+        if sure == 'y':
+            open(args.file, 'w').close()
+    elif args.check:
```

Comment:
I mean, I added this feature because I got annoyed at mine. I also added the extra hoops cause I figured this should be something that is hard to do.

If you think it's a bad idea, I can remove the feature.